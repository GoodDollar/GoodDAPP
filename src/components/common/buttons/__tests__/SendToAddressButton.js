import React from 'react'
import renderer from 'react-test-renderer'

import ImportedSendToAddressButton from '../SendToAddressButton'
import { withThemeAndLocalizationProvider } from '../../../../__tests__/__util__'
const SendToAddressButton = withThemeAndLocalizationProvider(ImportedSendToAddressButton)

jest.setTimeout(20000)

describe('SendToAddressButton', () => {
  it(`should match snapshot`, async () => {
    // Given
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<SendToAddressButton onPress={() => {}} />)))

    // When
    const tree = component.toJSON()

    // Then
    expect(tree).toMatchSnapshot()
  })
})
