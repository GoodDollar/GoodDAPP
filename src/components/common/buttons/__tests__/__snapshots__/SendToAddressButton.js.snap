// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendToAddressButton should match snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    data-focusable={true}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "MozTransitionProperty": "opacity",
        "MozUserSelect": "none",
        "WebkitTransitionDuration": "0.15s",
        "WebkitTransitionProperty": "opacity",
        "WebkitUserSelect": "none",
        "cursor": "pointer",
        "msTouchAction": "manipulation",
        "msUserSelect": "none",
        "touchAction": "manipulation",
        "transitionDuration": "0.15s",
        "transitionProperty": "opacity",
        "userSelect": "none",
      }
    }
    tabIndex="0"
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "alignItems": "center",
          "display": "flex",
          "marginLeft": "20px",
          "msFlexAlign": "center",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxPack": "center",
            "WebkitJustifyContent": "center",
            "alignItems": "center",
            "backgroundColor": "rgba(12,38,61,1.00)",
            "borderBottomLeftRadius": "21px",
            "borderBottomRightRadius": "21px",
            "borderTopLeftRadius": "21px",
            "borderTopRightRadius": "21px",
            "display": "flex",
            "height": "42px",
            "justifyContent": "center",
            "marginBottom": "5px",
            "msFlexAlign": "center",
            "msFlexPack": "center",
            "width": "42px",
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(255,255,255,1.00)",
              "fontFamily": "gooddollar",
              "fontSize": "26px",
              "fontStyle": "normal",
              "fontWeight": "normal",
            }
          }
        >
          î¡‚
        </div>
      </div>
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(12,38,61,1.00)",
            "direction": "ltr",
            "fontFamily": "Roboto",
            "fontSize": "10px",
            "fontWeight": "500",
            "lineHeight": "11px",
            "textAlign": "center",
            "textDecoration": "none",
            "textTransform": "none",
          }
        }
      >
        Send to address
      </div>
    </div>
  </div>
</div>
`;
